Fixes the issue where USE=cxx would cause the build to fail on creation of the
dynamic library for libxmlrpc_server_cgi++. That happens at least on OSX, but
maybe others as well. Was fixed in unstable 1.19.01 and will be in stable 1.20.

--- xmlrpc-c-1.18.02/src/cpp/Makefile.orig	2009-09-21 15:32:24 +0200
+++ xmlrpc-c-1.18.02/src/cpp/Makefile	2009-09-21 15:32:43 +0200
@@ -174,6 +174,8 @@ $(LIBXMLRPC_SERVER_ABYSSPP_SH): LIBDEP =
    -L$(LIBXMLRPC_ABYSS_DIR) -lxmlrpc_abyss \
    -L$(LIBXMLRPC_UTIL_DIR) -lxmlrpc_util \
 
+LIBXMLRPC_SERVER_CGIPP_SH = $(call shlibfn, libxmlrpc_server_cgi++)
+
 $(LIBXMLRPC_SERVER_CGIPP_SH): $(LIBXMLRPC_SERVER_CGIPP_MODS:%=%.osh) \
   $(call shliblefn, libxmlrpc_server++) \
   $(call shliblefn, lib$(LIBXMLRPCPP_NAME)) \
